l5kit/l5kit/dataset/ego.py

```diff
@@ -124,6 +147,7 @@ def get_frame(self, scene_index: int, state_index: int, track_id: Optional[int]
             "centroid": data["centroid"],
```

Comment:
nit: shall we also return host id? in case we would like to reduce metrics among neighboring frames (e.g. diff of open loop velocities / acceleration among neighboring frames) which requires car id in addition to timestamp